### ISSUE TITLE
Custom Border for Multiple lines right and left Border seems to have a bias

### ISSUE THREAD

**Issue by jupie** [simonbengtsson/jsPDF-AutoTable#920]
Hello, 

there seems to be a bias on the left and right Border if your print multiple cells. 

Code to reproduce
```
doc = new jsPDF();
    autoTable(this.doc,{
      body: [
![Screenshot 2022-09-22 192159](https://user-images.githubusercontent.com/17769900/191881338-ec69e0ad-0ccb-4afd-bced-24c5b89c4b51.png)

        [{content:'Cell 1', styles: { fillColor:[255,255,255],lineColor:[0,0,0,], lineWidth:{top: 1, right: 1, bottom: 0, left: 1} }},{ content: 'Test', rowSpan: 2,colSpan:2, styles: { halign:'center',valign: 'middle', fontSize: 18,fontStyle:'bold'  } }],
        [{content:'Cell 2 ', styles:  {fillColor:[255,255,255],lineColor:[0,0,0,], lineWidth:{top: 0, right: 1, bottom: 0, left: 1} }}],
        [{content: 'Cell3', styles: {fillColor:[255,255,255],lineColor:[0,0,0,], lineWidth:{top: 0, right: 0.5, bottom: 0.5, left: 1} }}]
      ],theme: 'plain'
    }); 

    this.doc.save('test.pdf');


```

Result: 

![Screenshot 2022-09-22 192159](https://user-images.githubusercontent.com/17769900/191881355-0c26a192-092e-4a32-b758-033af64cbfad.png)

**Comment by MikeDabrowski** [comment]
@jupie This should have been fixed in #961 and the colors perfected in #966 

```typescript
var doc = new jsPDF()
  doc.autoTable({
    body: [
      [{content:'Cell 1', styles: { fillColor:[255,255,255],lineColor:[0,0,0,], lineWidth:{top: 1, right: 1, bottom: 0, left: 1} }},{ content: 'Test', rowSpan: 2,colSpan:2, styles: { halign:'center',valign: 'middle', fontSize: 18,fontStyle:'bold'  } }],
      [{content:'Cell 2 ', styles:  {fillColor:[255,255,255],lineColor:[0,0,0,], lineWidth:{top: 0, right: 1, bottom: 0, left: 1} }}],
      [{content: 'Cell3', styles: {fillColor:[255,255,255],lineColor:[0,0,0,], lineWidth:{top: 0, right: 0.5, bottom: 0.5, left: 1} }}]
    ],
    tableWidth: 'wrap',
    theme: 'plain'
  })
  return doc
```

![image](https://github.com/simonbengtsson/jsPDF-AutoTable/assets/17836911/6e3781b2-36bd-46ed-832a-433c7bdddf97)

@simonbengtsson Imo this issue can be closed

**Comment by simonbengtsson** [comment]
Great! #966 is now released as well.
